movies/src/main/java/comp.fundacionjala/movies/Movie.java

```diff
@@ -0,0 +1,40 @@
+package comp.fundacionjala.movies;
+
+public class Movie {
+
+    private String title;
+    private int priceCode;
+    public static final int REGULAR_MOVIE = 0;
```

Comment:
Is it necessary to use CHILDREN_MOVIE, REGULAR_MOVIE, NEW_RELEASE_MOVIE, and priceCode variables ?
because now you are using inherintance